include/dynd/types/str_util.hpp

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (C) 2011-15 DyND Developers
+// BSD 2-Clause License, see LICENSE.txt
+//
+
+static inline bool is_simple_identifier_name(const std::string &s)
+{
```

Comment:
`static` should be removed here, that should only be used for functions in a `.cpp` file